### PR TITLE
release: v0.5.0

### PR DESCRIPTION
**Pull request by mniestroj**
See CHANGELOG.md for details.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,32 @@ All notable changes to this project will be documented in this file.
...
+## [0.5.0] - 2022-12-09
+### Added
+- Python library and CLI tools for accessing Golioth REST API (`scripts/python/golioth/`)
+- twister+pytest tests for `samples/rpc/`
+- support for `GOLIOTHCTL_CONFIG` environment variable in pytest scripts
+- `GOLIOTH_CIPHERSUITES` Kconfig option, which allows to select preferred ciphersuites used during
+  DTLS handshake (by default only one ciphersuite is selected to reduce DTLS handshake)
+
+### Fixed
+- handling of multiple desired firmware changes in `samples/dfu/`, which previously cause corrupted
```
Comment (szczys) [inline CHANGELOG.md:16]:
```suggestion
- handling of multiple desired firmware changes in `samples/dfu/`, which previously caused corrupted
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,32 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.5.0] - 2022-12-09
+### Added
+- Python library and CLI tools for accessing Golioth REST API (`scripts/python/golioth/`)
+- twister+pytest tests for `samples/rpc/`
+- support for `GOLIOTHCTL_CONFIG` environment variable in pytest scripts
+- `GOLIOTH_CIPHERSUITES` Kconfig option, which allows to select preferred ciphersuites used during
```
Comment (szczys) [inline CHANGELOG.md:12]:
```suggestion
- `GOLIOTH_CIPHERSUITES` Kconfig option, which allows selection of preferred ciphersuites used during
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,32 @@ All notable changes to this project will be documented in this file.
...
+  DTLS handshake (by default only one ciphersuite is selected to reduce DTLS handshake)
+
+### Fixed
+- handling of multiple desired firmware changes in `samples/dfu/`, which previously cause corrupted
+  firmware upgrade
+- infinite loop in ciphersuite negotiation during DTLS handshake, solved by narrowing down used
+  ciphersuites by introduced `GOLIOTH_CIPHERSUITES` Kconfig option
```
Comment (szczys) [inline CHANGELOG.md:19]:
```suggestion
  ciphersuites by introducing `GOLIOTH_CIPHERSUITES` Kconfig option
```

**Review by szczys — APPROVED** [review]
LGTM, I commented on some nits on the changelog.

**Comment by mniestroj** [comment]
@szczys Please recheck. Should be fixed.

**Review by szczys — APPROVED** [review]
LGTM!

File: README.rst [hunk 4]
```diff
@@ -100,59 +100,59 @@ During release testing, all boards and services are verified using sample applic
 |                  |              |             |Golioth   |
 |                  |              |             |version   |
 +==================+==============+=============+==========+
-|nRF91 Development |Authentication|v0.4.0       |v0.28.0   |
+|nRF91 Development |Authentication|v0.5.0       |v0.29.0   |
```
Comment (ncmiller) [inline README.rst:103]: Should update these to `v0.30.2`.

Comment (mniestroj) [reply README.rst]: Done.

**Review by ncmiller — APPROVED** [review]

**Pull request merged**